roles/satellite-clone/tasks/main.yml

```diff
@@ -120,6 +120,10 @@
 - name: Wait for foreman-tasks service to start
   pause: minutes=5
 
+- name: Run installer upgrade
+  command: satellite-installer --upgrade
+  when: satellite_version == 6.2
```

Comment:
@sthirugn I haven't had to do this step for 6.1, so I just made the change for 6.2